PWGHF/TableProducer/HFXToJpsiPiPiCandidateSelector.cxx

```diff
@@ -191,16 +193,21 @@ struct HFXToJpsiPiPiCandidateSelector {
       auto trackPos = hfCandX.index1_as<aod::BigTracksPID>(); //positive daughter
       auto trackNeg = hfCandX.index2_as<aod::BigTracksPID>(); //negative daughter
 
+      int selJpsiToEE = 1;
+      int selJpsiToMuMu = 1;
+
       // check if flagged as X --> Jpsi Pi Pi
-      if (!(hfCandX.hfflag() & 1 << XToJpsiPiPi)) {
-        hfSelXToJpsiPiPiCandidate(0);
-        // Printf("X candidate selection failed at hfflag check");
-        continue;
+      if (!(hfCandX.hfflag() & 1 << hf_cand_x::DecayType::XToJpsiToEEPiPi)) {
+        selJpsiToEE = 0;
+      }
+
+      if (!(hfCandX.hfflag() & 1 << hf_cand_x::DecayType::XToJpsiToMuMuPiPi)) {
+        selJpsiToMuMu = 0;
       }
 
```

Comment:
You should continue here if `(selJpsiToEE == 0 && selJpsiToMuMu == 0)`.

Reply:
Indeed! Now this is added